docs/tutorials/setup-development-machine-with-vagrant.md

```diff
@@ -187,7 +190,7 @@ vagrant destroy
 vagrant up
 ```
 
-## Deploy Tracee on Kubernetes
+## Deploy Tracee with Postee on Kubernetes
```

Comment:
While testing 0.7.0 I realized that we don't provide simple deployment YAML to install just Tracee. It's preconfigured with Postee webhook.